contrib/azure-setup.sh

```diff
@@ -49,13 +49,13 @@ showhelp() {
 requirements() {
     found=0
 
-    azureversion=$(azure -v)
+    azureversion=$(az -v)
     if [ $? -eq 0 ]; then
         found=$((found + 1))
         echo "Found azure-cli version: $azureversion"
     else
-        echo "azure-cli is missing. Please install azure-cli from"
-        echo "https://azure.microsoft.com/en-us/documentation/articles/xplat-cli-install/"
+        echo "azure cli is missing. Please install azure cli from"
```

Comment:
I would point people in the Azure portal, and ask them to use Cloud Shell.

Reply:
Sounds good, added verbiage and link for user to learn about the Cloud Shell